src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java

```diff
@@ -10,13 +10,13 @@
 
 
 /**
+ * Provides an implementation of the result set metadata to the SQL Server.
```

Comment:
Provides an implementation of <code>java.sql.ResultSetMetadata</code> JDBC Interface to assist in working with ResultSet metadata for SQL Server